MediaGalleryUi/view/adminhtml/web/template/image/image-edit.html

```diff
@@ -5,7 +5,7 @@
  */
 -->
 <div class="edit-image-details" if="image">
-    <form id="image-edit-details-form" method="post" enctype="multipart/form-data">
+    <form data-bind="mageInit:{'validation':{}}" id="image-edit-details-form" method="post" enctype="multipart/form-data">
```

Comment:
It would be better to use a shortcut for mage init:
```suggestion
    <form data-mage-init='{"validation":{}}' id="image-edit-details-form" method="post" enctype="multipart/form-data">
```

Reply:
@sivaschenko Thank you for reviewing! I tried using the shortcut but this gives me a console error. "Uncaught Error: cannot call methods on validation prior to initialization; attempted to call method 'isValid'".